Content.Server/Administration/Commands/AGhost.cs

```diff
@@ -1,72 +1,122 @@
-﻿using Content.Server.GameTicking;
+﻿using System.Linq;
+using Content.Server.GameTicking;
+using Content.Server.Ghost;
+using Content.Server.Mind;
 using Content.Shared.Administration;
 using Content.Shared.Ghost;
 using Content.Shared.Mind;
+using Robust.Server.GameObjects;
+using Robust.Server.Player;
 using Robust.Shared.Console;
 
-namespace Content.Server.Administration.Commands
+namespace Content.Server.Administration.Commands;
+
+[AdminCommand(AdminFlags.Admin)]
+public sealed class AGhost : LocalizedCommands
 {
-    [AdminCommand(AdminFlags.Admin)]
-    public sealed class AGhost : IConsoleCommand
+    [Dependency] private readonly IEntityManager _entities = default!;
+    [Dependency] private readonly IPlayerManager _playerManager = default!;
+
+    public override string Command => "aghost";
+    public override string Description => LocalizationManager.GetString("aghost-description");
+    public override string Help => "aghost";
+
+    public override CompletionResult GetCompletion(IConsoleShell shell, string[] args)
     {
-        [Dependency] private readonly IEntityManager _entities = default!;
+        if (args.Length == 1)
+        {
+            var names = _playerManager.Sessions.OrderBy(c => c.Name).Select(c => c.Name).ToArray();
+            return CompletionResult.FromHintOptions(names, "[Username]");
```

Comment:
Localising you absolute fool.

Reply:
> absolute fool

this is cyber bullying, i will never recover from this